pkg/operator/controller/status/controller.go

```diff
@@ -313,8 +331,24 @@ func checkAllIngressesAvailable(ingresses []operatorv1.IngressController) bool {
 	return len(ingresses) != 0
 }
 
-// computeOperatorDegradedCondition computes the operator's current Degraded status state.
-func computeOperatorDegradedCondition(ingresses []operatorv1.IngressController) configv1.ClusterOperatorStatusCondition {
+func computeOperatorDegradedCondition(ingresses []operatorv1.IngressController, unmanagedGatewayAPICRDNames string) configv1.ClusterOperatorStatusCondition {
```

Comment:
I would prefer to see this with it's own unique name, and called from the function formerly named `computeOperatorDegradedCondition`.